setup.cfg

```diff
@@ -53,8 +53,8 @@ install_requires =
 # The usage of test_requires is discouraged, see `Dependency Management` docs
 # tests_require = pytest; pytest-cov
 # Require a specific Python version, e.g. Python 2.7 or >= 3.4
-python_requires = >=3.6
```

Comment:
Moving this down a line looks like a mistake?

Reply:
Yeah, I am not sure what caused it, I'll make the change